validation-test/stdlib/ArrayTraps.swift.gyb

```diff
@@ -29,13 +29,30 @@ def bounds_trap(index, expr_to_write):
 % for ArrayTy in array_types:
 
 var ${ArrayTy}Traps = TestSuite("${ArrayTy}Traps" + testSuiteSuffix)
+var ${ArrayTy}_lower_bound_out_of_range_error_msg = (
+% if ArrayTy in ['Array', 'ContiguousArray']:
+    "Negative ${ArrayTy} index is out of range"
+% else:
+    "${ArrayTy} index is out of range (before startIndex)"
+% end
+)
+
+var ${ArrayTy}_bounds_error_msg = (
+% if ArrayTy in ['Array', 'ContiguousArray']:
+    "Index out of range"
+% else:
+    "Index out of bounds"
```

Comment:
Seems like an unimportant difference in messages between these types -- it would make sense to standardize on one.  (As a separate PR.)